### PR TITLE
(DOCSP-28448) [OM/CM] Managed Sharded Collection EOL banner

### PR DESCRIPTION
**Pull request by zach-carr**
[Jira](https://jira.mongodb.org/browse/DOCSP-28448)

Will be staged with mms-docs, but this part has to be merged into docs-tools first

Using [this wiki](https://wiki.corp.mongodb.com/pages/viewpage.action?pageId=149195138)

Rephrased the copy from the ticket
> CM docs message - 
"On June 16h, 2023, we will be ending SNMP and Manage Sharded Collections UI support for MongoDB in Cloud Manager. We will remove these features from the UI completely, at a later date. The removal of Manage Sharded Collections UI does not cause any impact to your sharded clusters."
OM docs message -
"In Ops Manager 7.0, we will be removing SNMP and Manage Sharded Collections UI support for MongoDB in Ops Manager. The removal of Manage Sharded Collections UI does not cause any impact to your sharded clusters."

### CODE REVIEW

File: themes/mms-onprem/page.html [hunk 1]
```diff
@@ -87,19 +87,35 @@ <h4 class="alert-heading" style="margin-bottom: 10px;">
...
+      </button>
+      <span class="alert-message">
+        <h4 class="alert-heading" style="margin-bottom: 10px;">
+          Support for Ops Manager 5.0 Ends July 2023.
+        </h4>
+  
+        <a href="{{theme_base_url}}/ops-manager/v6.0/tutorial/upgrade-ops-manager/">Upgrade Ops Manager to version 6.0</a> for continued support.
+      </span>
+    </div>
+  {%- endif %}
```
Comment (zach-carr) [inline themes/mms-onprem/page.html:102]: unrelated indentation fix, nothing else changed

File: themes/mms-cloud/theme.conf [hunk 2]
```diff
@@ -16,4 +16,5 @@ edition = EDITION
 nav_excluded = NAV
 automation_eol =
 backup_monitoring_eol_past =
+managed_sharded_collection_eol =
```
Comment (zach-carr) [inline themes/mms-cloud/theme.conf:19]: note: this is supposed to be empty

File: themes/mms-onprem/page.html [hunk 3]
```diff
@@ -87,19 +87,35 @@ <h4 class="alert-heading" style="margin-bottom: 10px;">
...
+
+  {%- if theme_managed_sharded_collection_eol %}
+    <div class="alert alert-info alert-dismissible fade in" role="alert" style="margin-left: 20px">
+      <button type="button" class="close" data-dismiss="alert" aria-label="Close">
+        <span aria-hidden="true">&times;</span>
+      </button>
+      <span class="alert-message">
+        <h4 class="alert-heading" style="margin-bottom: 10px;">
+          In version 7.0, MongoDB Ops Manager will end support for SNMP and
```
Comment (melissamahoney-mongodb) [inline themes/mms-onprem/page.html:111]: Question for tech review: Do we have a date for v7.0? I think that'd be more useful here than an ambiguous future version.

**Review by melissamahoney-mongodb — APPROVED** [review]
LGTM, mod one question for tech review.
